src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java

```diff
@@ -21,32 +22,37 @@
 
     @Override
     @PreAuthorize("hasPermission({'tenant':#tenant}, 'ACTIVATION.TENANT.CREATE')")
+    @PrivilegeDescription("Privilege to add a new activation tenant")
     public ResponseEntity<Void> addTenant(Tenant tenant) {
         tenantService.createTenant(tenant.getTenantKey());
         return ResponseEntity.ok().build();
     }
 
     @Override
     @PreAuthorize("hasPermission({'tenantKey':#tenantKey}, 'ACTIVATION.TENANT.DELETE')")
+    @PrivilegeDescription("Privilege to delete activation tenant")
     public ResponseEntity<Void> deleteTenant(String tenantKey) {
         tenantService.deleteTenant(tenantKey);
         return ResponseEntity.ok().build();
     }
 
     @Override
     @PostAuthorize("hasPermission(null, 'ACTIVATION.TENANT.GET_LIST')")
+    @PrivilegeDescription("Privilege to get all activation tenants")
     public ResponseEntity<List<Tenant>> getAllTenantInfo() {
         throw new UnsupportedOperationException();
     }
 
     @Override
     @PostAuthorize("hasPermission({'returnObject': returnObject.body}, 'ACTIVATION.TENANT.GET_LIST.ITEM')")
+    @PrivilegeDescription("Privilege to get activation tenant")
```

Comment:
suggest description:
`Privilege to get tenant by id on activation micro-service`